libs/platform/experience/src/services/registry/default-components-registry.service.ts

```diff
@@ -32,7 +32,14 @@ export class DefaultComponentsRegistryService
     uid: string,
     styleClasses?: string
   ): TemplateResult | undefined {
-    const component = this.componentMapping?.[type];
+    let component;
```

Comment:
component mappig should have precedence, and only if there is no explicit mapping defined, we want to reach to component plugin, to check, if it provides this component.

Reply:
but actually, let's simplify:
- don't check component plugin at all (bring back protected method)
- just render the type as tag always, even if we don't have a mapping